x-pack/plugins/observability/server/services/slo/get_slo.ts

```diff
@@ -26,6 +31,12 @@ export class GetSLO {
       time_window: slo.time_window,
       budgeting_method: slo.budgeting_method,
       objective: slo.objective,
+      summary: {
```

Comment:
@kdelemme, would please help me understand the return of `summary` key while I consumed nothing from my error budget, why initial is 0.05
`"summary":{"sli_value":0,"error_budget":{"initial":0.05,"consumed":0,"remaining":1}}`

Reply:
`initial` is the initial value of your error budget for the period, defined as `1 - SLO target`, e.g. for a 95% SLO, the error budget is `1 - 0.95 = 0.05` or 5%.
`consumed` and `remaining` are relative to that value, e.g. `consumed = 0.1` means you consumed 10% of your error budget.

Reply:
That make sense, thanks!